cg/models/taxprofiler/taxprofiler.py

```diff
@@ -1,9 +1,20 @@
 from pathlib import Path
 
-from pydantic.v1 import Field
+from pydantic.v1 import Field, BaseModel
```

Comment:
```suggestion
from pydantic mport Field, BaseModel
```
Make sure to use pydantic v2 instead of v1, we are trying to migrate way from v1.